lib/working_hours/computation.rb

```diff
@@ -101,8 +101,7 @@ def advance_to_closing_time time, config: nil
         time_in_day = time.seconds_since_midnight
         time = time.beginning_of_day
         config[:working_hours][time.wday].each do |from, to|
-          return time + to if time_in_day >= from and time_in_day < to
-          return time + to if from >= time_in_day
+          return time + to if time_in_day < to
```

Comment:
This one I just simplified, nothing to do with precision I just noticed it was unnecessarily complicated (likely because it was copied from `advance_to_working_time`)